docs/src/components/Layout/Sidebar.tsx

```diff
@@ -199,26 +202,33 @@ const SecondaryNav = (props) => {
         </ExpanderItem>
       )}
 
-      <ExpanderItem
-        title={<ExpanderTitle Icon={MdOutlineAutoAwesome} text="Theming" />}
-        value="theming"
-      >
-        {theming.map(({ label, ...rest }) => (
-          <NavLink key={label} {...rest} onClick={props.onClick}>
-            {label}
-          </NavLink>
-        ))}
-      </ExpanderItem>
-      <ExpanderItem
-        title={<ExpanderTitle Icon={MdOutlineArticle} text="Guides" />}
-        value="guides"
-      >
-        {guides.map(({ label, ...rest }) => (
-          <NavLink {...rest} key={label} onClick={props.onClick}>
-            {label}
-          </NavLink>
-        ))}
-      </ExpanderItem>
+      {/* React Native does not yet support theming */}
+      {isReactNative ? null : (
```

Comment:
Other nit, for readability,  is I typically would prefer having a variable like `{themingEnabled && <Component` and defining at the top what the rendering conditions are, rather than having to dig through these sections to figure out which platform supports theming.

Reply:
Just trying to follow existing patterns

Reply:
Non consequential, we'll be adding these pages at some point anyway.